recipes/nvcc/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "nvcc" %}
+{% set version = environ.get("CUDA_VER", "9.2") %}
```

Comment:
Can you add a `recipe/conda_build_config.yaml` with,

```yaml
CUDA_VER:
  - 9.2
  - 10.0
```
and use the value like `set version = CUDA_VER`

Reply:
The `9.2` is really only here to make the linter happy. Personally I'd rather not have it here at all if it could be avoided.

Basically what this is doing is picking up `CUDA_VER` from the docker image.